lfs/setup.go

```diff
@@ -28,11 +28,13 @@ var (
 		Properties: map[string]string{
 			"clean":    "git-lfs clean -- %f",
 			"smudge":   "git-lfs smudge -- %f",
+			"process":  "git-lfs filter",
```

Comment:
Good call. I think `process` is a little generic, but I'm on board with `git lfs filter-process`.